File: RegexLFA/RgxReplace.cs
using System.Text.RegularExpressions;

namespace RegexLFA;

internal class RgxReplace
{
    public static void GetRgxReplace()
    {
        // Uma string longa com toneladas de espaços em branco  
        string comEspaco = "Aqui    está uma   string com muitos    espaços em branco.";
        Console.WriteLine($" Com espaço: {comEspaco}\n");

        string semEspaco = Regex.Replace(comEspaco, "\\s+", "");
        Console.WriteLine($" Sem espaço: {semEspaco}");
    }
}

File: RegexLFA.VBNET/RgxReplace.vb
Imports System.Text.RegularExpressions

Public Class RgxReplace
    Public Shared Sub GetRgxReplace()
        Dim comEspaco = "Aqui    está uma   string com muitos    espaços em branco."
        Console.WriteLine($" Com espaço: {comEspaco}")

        Dim semEspaco = Regex.Replace(comEspaco, "\s+", "")
        Console.WriteLine($" Sem espaço: {semEspaco}")
    End Sub
End Class

